### PR TITLE
drop workerTasks without startTime field

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] Automatic tests added/update
- [x] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

### CODE REVIEW

File: src/migrations/migration_helpers.js [hunk 1]
```diff
@@ -45,4 +51,4 @@ const deduplicateDocuments = async (db, collectionName, uniqueKey) => {
   }
 };
 
-export {collectionExists, safeDropIndex, deduplicateDocuments, safeCreateUniqueIndex};
```
Comment (krzysztof-jelski) [inline src/migrations/migration_helpers.js]: We can stop exporting `safeDropIndex`, as it is used only in migration_helpers now.

File: src/migrations/20181218115000_worker_task_ttl.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {safeDropIndex} from './migration_helpers';
 
 // eslint-disable-next-line import/prefer-default-export
 export const up = async (db, {workerTasksTTLInSeconds}, logger) => {
+  await db.collection('workerTasks').remove({startTime: {$exists: false}});
```
Comment (spherefoundry) [inline src/migrations/20181218115000_worker_task_ttl.js]:
```suggestion
  await db.collection('workerTasks').deleteMany({startTime: {$exists: false}});
```

File: src/migrations/20181218154000_worker_index_ttl.js [hunk 3]
```diff
@@ -7,11 +7,11 @@ This Source Code Form is subject to the terms of the Mozilla Public License, v.
 This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
 */
 
-import {safeDropIndex} from './migration_helpers';
+import {safeDropCollection} from './migration_helpers';
 
 // eslint-disable-next-line import/prefer-default-export
 export const up = async (db, {workerLogsTTLInSeconds}, logger) => {
-  await safeDropIndex(db, 'workerLogs', 'timestamp_-1');
+  await safeDropCollection(db, 'workerLogs');
```
Comment (spherefoundry) [inline src/migrations/20181218154000_worker_index_ttl.js]: IMO: this goes too far.

File: src/migrations/20181218115000_worker_task_ttl.js (hunk 2 above)
Comment (spherefoundry) [reply src/migrations/20181218115000_worker_task_ttl.js]: Also, this line should be a separate migration. Moving a existing migration to the end of the queue, is not a good idea. Other migrations written since, may depend on it being run before them.
